app/controllers/katello/api/v2/content_view_puppet_modules_controller.rb

```diff
@@ -80,7 +76,7 @@ def find_content_view_puppet_module
     end
 
     def find_puppet_module
-      @puppet_module = PuppetModule.find(params[:id]) if params[:id]
+      @puppet_module = PuppetModule.find_by(puppet_module_params)
```

Comment:
@ehelms If a user adds a puppet module by puppet module ID, should we set the uuid or name/author?